pkg/plugins/builtin/http/http.go

```diff
@@ -132,6 +134,20 @@ func exec(stepName string, config interface{}, ctx interface{}) (interface{}, in
 		return nil, nil, fmt.Errorf("HTTP request failed: %s", err.Error())
 	}
 
+	// remove response magic prefix
+	if cfg.TrimPrefix != "" {
+		trimPrefixBytes := []byte(cfg.TrimPrefix)
+		respBody, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return nil, nil, fmt.Errorf("HTTP cannot read response: %s", err.Error())
+		}
+		resp.Body.Close()
+		if bytes.HasPrefix(respBody, trimPrefixBytes) {
```

Comment:
```suggestion
		respBody = bytes.TrimPrefix(respBody, trimPrefixBytes)
```

Reply:
My suggestion concerned 3 lines, unsure it's gonna merge correctly if applied as-is